app/models/like.rb

```diff
@@ -1,5 +1,5 @@
 class Like < ActiveRecord::Base
-  belongs_to :author, class_name: 'User', foreign_key: :user_id
+  belongs_to :user, class_name: 'User', foreign_key: :user_id
   belongs_to :post
 
```

Comment:
- [ ] LikesCounter must be an integer greater than or equal to zero, as informed in the previous review.

Reply:
This hs been resolved.